sdk/costmanagement/Azure.ResourceManager.CostManagement/src/autorest.md

```diff
@@ -89,6 +96,24 @@ rename-mapping:
   Scope: BenefitRecommendationScope
   Grain: BenefitRecommendationUsageGrain
   OperatorType: ComparisonOperatorType
+  DismissAlertPayload.properties.closeTime: -|datetime
```

Comment:
consider renaming to CloseOn to align with others

Reply:
autorest will auto rename all datetime to `FooOn`